### DISCUSSION TITLE
Minor amends to the payment pop-up (Pay with, QR icon, amount)

### DISCUSSION THREAD

**Discussion by BenGWeeks** [btcpayserver/btcpayserver#3670] [Ideas / Feature Requests]
Hi there,

Could you please consider the following amendments to the payment pop-up:

![BTCPayServer Mockup](https://user-images.githubusercontent.com/5950171/164545105-aea79f8a-43e2-41c9-9a8d-2bd875a773f5.png)

1. Instead of a "pay with" being a drop-down, we use highlighted buttons
2. We enter the amount they are paying in the default currency (not just the BTC value)
3. Instead of the shop name, we have "Due" (optionally move the shop name to the top next to the icon)
4. We put an easily recognisable Lightning icon in the QR code when Lightning is selected as a payment method

NB I noticed that you actually offer LNUrl payments, do people use that for paying specific amounts though?

Many thanks.

Ben

**Comment by BenGWeeks** [comment]
Would this be of interest @Kukks and @pavlenex? Might be a good one for me to make a pull request on?

**Reply by pavlenex** [reply]
No need to do a PR, we'll work on completely re-designing this page in 1.6.0-1.7.0 you can follow the progress [here](https://github.com/btcpayserver/btcpayserver/issues/3567). Not sure it'll address all of your comments but we'll take them into consideration as we're thinking on how we may want to re-design this customer-facing page, since it's the only page an end-user is interacting with, it takes quite a bit of thinking and research before we re-design it.

**Reply by BenGWeeks** [reply]
Agreed. Looking forward to it! :-)

**Comment by Kukks** [comment]
I would suggest we add a toggle between current format and the new redesign
@pavlenex to ensure checkouts don't get broken (custom css, plugins, etc)
